src/components/Search.tsx

```diff
@@ -21,10 +31,10 @@ type SearchBarProps = {
 
 const SearchBar = ( { handleTextChanges }: SearchBarProps ) => (
   <View style={styles.searchBar}>
-    <Icon name="magnify" size={25} />
+    <Icon name="magnify" size={25} style={styles.centered} />
     <TextInput
       placeholder="Koj"
```

Comment:
@bhajneet do we want to bake in internationalisation from the start? This would be an example of where to start.

Reply:
Related #39

Reply:
Yes absolutely, how can I help?

Reply:
I think search screen cannot be a candidate yet since we will only support Punjabi search. Having that screen internationalized won’t make much sense unless we figure out how to convert search into languages we support with internationalization.

Reply:
The placeholder can be "Search" or "Khoj" or whatever language is chosen.